usr/src/uts/common/fs/zfs/sys/uberblock_impl.h

```diff
@@ -54,6 +55,12 @@ struct uberblock {
 
 	/* highest SPA_VERSION supported by software that wrote this txg */
 	uint64_t	ub_software_version;
+
+	/* These fields are reserved for features that are under development: */
+	uint64_t	ub_mmp_magic;
+	uint64_t	ub_mmp_delay;
```

Comment:
[ Copying a question here I posted in the RTI forum. ]
I'm not normally at this depth in ZFS, so let me ask:
How do we know this works with older on-disk data?